Simplified/Utilities/CwlDispatch.swift

```diff
@@ -0,0 +1,151 @@
+//
+//  CwlDispatch.swift
+//  CwlUtils
+//
+//  Created by Matt Gallagher on 2016/07/29.
+//  Copyright © 2016 Matt Gallagher ( https://www.cocoawithlove.com ). All rights reserved.
+//
+//  Permission to use, copy, modify, and/or distribute this software for any
+//  purpose with or without fee is hereby granted, provided that the above
+//  copyright notice and this permission notice appear in all copies.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+//  WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+//  MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY
+//  SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+//  WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+//  ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR
+//  IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+//
+
+import Foundation
+
+public extension DispatchSource {
+  // An overload of timer that immediately sets the handler and schedules the timer
+  class func singleTimer(interval: DispatchTimeInterval, leeway: DispatchTimeInterval = .nanoseconds(0), queue: DispatchQueue, handler: @escaping () -> Void) -> DispatchSourceTimer {
+    let result = DispatchSource.makeTimerSource(queue: queue)
+    result.setEventHandler(handler: handler)
+    #if swift(>=4)
```

Comment:
since we only support swift 5 and we are checking in this code, I wonder if we should just remove the swift < 5 cases to simplify reading this file. What do you think?